src/modules/user-auth/components/DefaultEmailUnverifiedPrompt.tsx

```diff
@@ -0,0 +1,5 @@
+import { VFC } from 'react';
+
+const DefaultEmailUnverifiedPrompt: VFC = () => <p>Email unverified</p>;
```

Comment:
Can we just have one `ErrorPageLayout` and pass in a message?

Reply:
Yea, we can do this later. This is beyond the scope of this issue.

Reply:
I mean instead of having DefaultEmailUnverified and DefaultUnauthorized

Reply:
I can imagine how the layout for these 2 pages can be different tho... But I just leave them there for placeholder anyway, we can change this later (removing them altogether is okay too).